File: Laboratorio/parpadeo.c
const int ledContinuo = 2;
const int ledTresVeces = 4;
int contador = 0;

void setup() {
  pinMode(ledContinuo, OUTPUT);
  pinMode(ledTresVeces, OUTPUT);
}

void loop() {
  digitalWrite(ledContinuo, HIGH);
  
  if (contador < 3) {
    digitalWrite(ledTresVeces, HIGH);
  }
  
  delay(500);
  
  digitalWrite(ledContinuo, LOW);
  
  if (contador < 3) {
    digitalWrite(ledTresVeces, LOW);
  }
  
  delay(500);
  
  if (contador < 3) {
    contador++;
  }
}

File: Laboratorio/parpadeo.py
from machine import Pin
import time

led_continuo = Pin(2, Pin.OUT)
led_tres_veces = Pin(4, Pin.OUT)
contador = 0

while True:

    led_continuo.value(1)
    
    if contador < 3:
        led_tres_veces.value(1)
        
    time.sleep(0.5)
    
    led_continuo.value(0)
    
    if contador < 3:
        led_tres_veces.value(0)
        
    time.sleep(0.5)
    
    if contador < 3:
        contador += 1 # Sumamos un ciclo
